crate/operator/data/cratedb_cluster_last_activity-collector.yaml

```diff
@@ -0,0 +1,14 @@
+---
+collector_name: cratedb_cluster_last_activity_collector
```

Comment:
I wonder if we should call it `cratedb_cluster_last_user_activity` to clearly state that this is only for user-initiated stuff and not system stuff...